src/stores/AccountController.ts

```diff
@@ -228,7 +228,10 @@ export const updatePersonalShelves = async () => {
   });
 };
 
-export const updateConsents = async (customerConsents: CustomerConsent[]): Promise<ServiceResponse<CustomerConsent[]>> => {
+export const updateConsents = async (
+  customerConsents: CustomerConsent[],
+  consentValues: Record<string, string | boolean>,
```

Comment:
Again, isn't this just the same data as `customerConsents` in a different format?